types/config.d.ts

```diff
@@ -12,6 +12,7 @@ export interface Config {
   asyncUtilTimeout: number
   computedStyleSupportsPseudoElements: boolean
   defaultHidden: boolean
+  defaultIgnore: string
```

Comment:
Could be in a follow-up, but I think it would be helpful if we add some JS-Docs to the types.
Currently the text `default value for the `ignore` option in `ByText` queries` isn't visible to the users.

Reply:
Added a comment here: https://github.com/testing-library/dom-testing-library/pull/1138/files#diff-19584312cba053f9726eeb4b0766364b873cd974e552656c557ae30ee45f9d00R15

Is that what you were looking for?